drivers/sensor/bh1750/bh1750.h

```diff
@@ -0,0 +1,42 @@
+/* bh1750.h - Driver for BH1750 ambient light sensor */
+/*
+ * Copyright (c) 2023 Sngular People SL.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_BH1750_H_
+#define ZEPHYR_DRIVERS_SENSOR_BH1750_H_
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/i2c.h>
+
+#define BH1750_POWER_DOWN		0x00
+#define BH1750_POWER_ON			0x01
+#define BH1750_RESET			0x07
+
+#define BH1750_CHANGE_INT_TIME_H_BIT	0x40
+#define BH1750_CHANGE_INT_TIME_L_BIT	0x60
+
+enum bh1750_resolution {
+	BH1750_H_RESOLUTION_MODE = 0,
+	BH1750_H_RESOLUTION_MODE2 = 1,
+	BH1750_L_RESOLUTION_MODE = 3
+};
+
+enum bh1750_mode {
+	BH1750_MODE_CONTINUOUSLY = 0,
+	BH1750_MODE_ONE_TIME = 1
+};
+
+struct bh1750_data {
+	float ambient_light;
+};
+
+struct bh1750_config {
+	struct i2c_dt_spec i2c;
+	enum bh1750_resolution resolution;
+	enum bh1750_mode mode;
+};
```

Comment:
I don't think it's worth having a separate private include file for this. Please move to `bh1750.c`

Reply:
Ok, I think so too.